### ISSUE TITLE
Enhancement: Make adding members quicker when on-site at the hash

### ISSUE THREAD

**Issue by diabWH3** [James-A-White/HarrierCentral#109]
Currently, adding new hashers/members with the app is tedious while on-site at the hash; the email address requirement is particularly time-consuming. There's a lot going on and usually lots of people trying to pay at the same time, and it's just generally a somewhat stressful time. I am finding myself adding people as "visitors" instead of taking the time to get their email address and real name to add them on the app. Then, because of their "visitor" status, keeping track of attendance becomes a chore across multiple runs.

Is there some better way to manage adding members ad-hoc? I realize the email address is used as a unique identifier in the database, so this could be a significant request. Our kennel isn't even that big, so I could imagine other larger kennels encountering the same scenario.

This is tangentially related to #55  because, even if I add someone as a real member with an accurate email address, they have no way of knowing I did that and nothing to remind them about downloading the app when they get home. A "welcome" email to the new user's address would be incredibly useful and I've already had multiple hashers ask me about a similar feature.

**Comment by James-A-White** [comment]
Excellent feedback. One alternative would be to do this: As a HC admin, you would present the user with a QR code that they could scan. It would then bring them to a webpage where they would enter their own details either as a visitor or for a new account. They would then show up in your run list accordingly.  Would this be a better solution?  OnOn!  Op

**Comment by James-A-White** [comment]
Yes... In fact it will be in the app and you would print it from there. I'll make that one of the next things I work on.

**Comment by diabWH3** [comment]
Sounds great!
